### PR TITLE
Updates to release plan, ext & issue template

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_bug_report.md [hunk 1]
```diff
@@ -9,7 +9,6 @@ labels: bug
 ## Environment data
 
 -   VS Code version: XXX
--   Extension version (available under the Extensions sidebar): XXX
```
Comment (DonJayamanne) [inline .github/ISSUE_TEMPLATE/1_bug_report.md:12]: Duplicate entry

File: .github/release_plan.md [hunk 2]
```diff
@@ -57,12 +57,11 @@
 
 ## Release
 
--   [ ] Publish the release via Azure DevOps
-    -   [ ] Make sure [CI](https://github.com/Microsoft/vscode-jupyter/blob/main/CONTRIBUTING.md) is passing. Try a re-run on any failing CI test stages. If tests still won't pass contact the owning team.
-    -   [ ] On Actions tab of GitHub, ensure a workflow has automatically run for the workflow `Release VSIX`.
-    -   [ ] Download the [Release VSIX](https://pvsc.blob.core.windows.net/extension-builds-jupyter/ms-toolsai-jupyter-release.vsix) & Make sure no extraneous files are being included in the `.vsix` file (make sure to check for hidden files)
-    -   [ ] Go into the `Publish Extension` workflow and manually run this workflow to publish the extension to the marketplace.
-    -   [ ] From a VSCode instance uninstall the python extension. After the publish see if the new version is available from the extensions tab. Download it and quick sanity check to make sure the extension loads.
+-   [ ] Publish the release manually
```
Comment (DonJayamanne) [inline .github/release_plan.md:60]: This is a manual step for now (GitHub actions support this, but we haven't enabled it for now)
I guess we can add support later.

File: package.json [hunk 3]
```diff
@@ -3,10 +3,6 @@
     "displayName": "Jupyter",
     "version": "2020.12.0-dev",
     "description": "Jupyter notebook support, interactive programming and computing that supports Intellisense, debugging and more.",
-    "featureFlags": {
-        "usingNewInterpreterStorage": true
-    },
-    "languageServerVersion": "0.5.30",
```
Comment (DonJayamanne) [inline package.json:9]: Not required, these are specific to the python extension

File: .github/ISSUE_TEMPLATE/1_bug_report.md (hunk 1 above)
Comment (IanMatthewHuff) [reply .github/ISSUE_TEMPLATE/1_bug_report.md:12]: Question, we are separate, but given our shared scenarios should this entry be changed to:
- Python Extension version (if installed): XXX
and put it below the Jupyter Extension version?

**Review by IanMatthewHuff — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/1_bug_report.md (hunk 1 above)
Comment (DonJayamanne) [reply .github/ISSUE_TEMPLATE/1_bug_report.md:12]: Done.

**Pull request merged**